src/plays/airbnb-experiences-clone/components/Card.jsx

```diff
@@ -0,0 +1,43 @@
+import katieZaferesImg from '../assets/images/katie-zaferes.png';
+import weddingImage from '../assets/images/wedding-photography.png';
+import mountainBikeImage from '../assets/images/mountain-bike.png';
+import starImage from '../assets/images/star.png';
+
+export default function Card(props) {
+  let badgeText;
+  if (props.openSpots === 0) {
+    badgeText = 'SOLD OUT';
+  } else if (props.location === 'Online') {
+    badgeText = 'ONLINE';
+  }
+
+  let cardImg;
+  let altText;
+  if (props.coverImg === 'katie-zaferes.png') {
+    cardImg = katieZaferesImg;
+    altText = 'Katie Zaferes';
+  } else if (props.coverImg === 'wedding-photography.png') {
+    cardImg = weddingImage;
+    altText = 'Woman wearing a white gown';
+  } else if (props.coverImg === 'mountain-bike.png') {
+    cardImg = mountainBikeImage;
+    altText = 'Mountain bike';
+  }
```

Comment:
As we are deriving the value from the props, can we use the useMemo here, therefore it will not recreate the variables on every render.

Reply:
Sure. Will use useMemo() instead.